.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,7 +6,6 @@
 1. [ ] Branch follows the naming format (`iox-123-this-is-a-branch`)
 1. [ ] Commits messages are according to this [guideline][commit-guidelines]
     - [ ] Commit messages have the issue ID (`iox-#123 commit text`)
-    - [ ] Commit messages are signed (`git commit -s`)
```

Comment:
@FerdinandSpitzschnueffler Why did you remove this item? Has the Eclipse policy changed?

Reply:
Yes, please see [this comment](https://github.com/eclipse-iceoryx/iceoryx/pull/1737#issuecomment-1277466399).